workflows/argo-cd/Workflow_Install ArgoCD.yaml

```diff
@@ -243,9 +243,11 @@ spec:
   - type: xlrelease.StringVariable
     key: runAsUsername
     showOnReleaseStart: false
+    description: Run automated tasks as this user
```

Comment:
Still missing label that is there in other places for these run as variables.

<img width="555" alt="Screenshot 2023-09-26 at 4 09 27 PM" src="https://github.com/digital-ai/release-content/assets/7783488/0d131acb-8b46-47b6-9fe0-2727f9dce2f8">

Reply:
Added the missing label in Argo workflows

Reply:
@isaacsatish Can you please scan all the places and follow a common standard for description?

- Run automated tasks as this user
- Password for Run Automated Tasks As user

Notice the difference in case Run automated tasks vs Password for Run **Automated** **Tasks**

I think more appropriate label should and description should be:
Label for `runAsUsername` variable:  **Run automated tasks as user**
Description for `runAsUsername` variable: **Run all automated tasks in this workflow as user**

Label for `runAsUserPassword` variable:  **Password**
Description for `runAsUserPassword` variable: **Password for automated tasks user**

Check with PM if whatever I suggested is good enough.